package.json

```diff
@@ -31,6 +31,6 @@
 		"gulp-prompt": "^1.2.0",
 		"semver": "^7.3.2",
 		"tslint": "~6.1.0",
-		"typescript": "~4.0.7"
+		"typescript": "~4.3.4"
```

Comment:
This shouldn't change unless strictly necessary, it's out of the scope for this PR